### PR TITLE
Add container reference to calculate distance from it

### PR DESCRIPTION
**Pull request by KlonD90**
Hi! I add container ref to calculate offset in right way if we have elements that not directly depends on window

### CODE REVIEW

File: src/useResizable.ts [hunk 1]
```diff
@@ -45,16 +46,26 @@ const useResizable = ({
...
+            return reverse ? left - e.clientX : e.clientX - left;
+          }
           return reverse ? document.body.offsetWidth - e.clientX : e.clientX;
         }
+        if (containerRef && containerRef.current) {
+          const containerNode = containerRef.current;
+          const { top } = containerNode.getBoundingClientRect();
+          return reverse ? top - e.clientY : e.clientY - top;
+        }
         return reverse ? document.body.offsetHeight - e.clientY : e.clientY;
```
Comment (RyoSogawa) [inline src/useResizable.ts:61]: @KlonD90 
This part doesn't seem to work well when `reverse === true`.
I think the size of the container element needs to be calculated.
```ts
return reverse ? height + top - e.clientY : e.clientY - top;
```

File: stories/AxisYWithContainer.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <Resizable {...props} containerRef={containerRef}>
+          {({ position: y, separatorProps }) => (
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column',
+                height: '100vh',
+                overflow: 'hidden',
+              }}
+            >
```
Comment (RyoSogawa) [inline stories/AxisYWithContainer.stories.tsx:30]: @KlonD90 
I know it's a small detail, but can I ask you to make sure the element doesn't overflow?
```suggestion
            <div
              style={{
                display: 'flex',
                flexDirection: 'column',
                height: 'calc(100vh - 32px)',
                overflow: 'hidden',
              }}
            >
```

**Review by RyoSogawa — CHANGES_REQUESTED** [review]
@KlonD90 
Thanks for the very useful features. ❤️ 
I've commented on a few points and hope you'll check them out.

File: src/useResizable.ts (hunk 1 above)
Comment (KlonD90) [reply src/useResizable.ts]: Thank you so much for feedback!

**Review by RyoSogawa — APPROVED** [review]
@KlonD90 
Thank you very much for your contribution! ❤️

**Pull request merged**